descriptors/ModuleDescriptor-template.json

```diff
@@ -237,7 +237,8 @@
           "pathPattern": "/orders/pieces",
           "permissionsRequired": ["orders.pieces.item.post"],
           "modulePermissions": [
-            "orders-storage.pieces.item.post"
+            "orders-storage.pieces.item.post",
+            "orders-storage.pieces.item.get"
```

Comment:
Probably this one was supposed to go to `"PUT"` section?

Reply:
Oh, I missed. Fixed.